### DISCUSSION TITLE
Shared albums does not show up

### DISCUSSION THREAD

**Discussion by Uberpainer** [immich-app/immich#4704] [General]
Hi,

I'm not able to figure out if this is a bug or a missing feature in the iOS app.

I have created a shared album on my account named "Asger og Alfred" and shared the album with my wifes account. When my wife log in from the webbrowser are she able to locate and see the album. But looking for the shared album from her device/mobile, then is it not visible.

I have tried log her out and back in on her account and even re-installed the app.

Can anyone help me if shared album from app still not are possible?

App version: 1.83.0 build.123
Server version: 1.83.0

![IMG_4332](https://github.com/immich-app/immich/assets/52506491/cb7000ef-df83-4936-95a6-dd0b4dac05ac)
![IMG_C4DD6657FD86-1](https://github.com/immich-app/immich/assets/52506491/661840f2-e430-4b3d-83fc-58290f2a7d05)

Regards
Anders Jensen

**Comment by Uberpainer** [comment]
Just did a second re-install of the app and the album did show up on my wife mobile. Something seams not to be running smooth.

**Reply by alextran1502** [reply]
On the mobile app, especially on the iPhone, the phone has to go through the process of hashing the assets first before the albums start to come up. 

The implementation is in Dart, so it is not the fastest. We plan to optimize this hashing algorithm to use Swift to increase the performance.

**Reply by Uberpainer** [reply]
Thank you for your quick reply as always 👍 
Out of all tried self-hosted solutions, then is Immich really a diamond between all others. Keep up the good work.

**Reply by alextran1502** [reply]
No problem! Cheers!
